llvm/test/CodeGen/AMDGPU/inlineasm-16-true16.ll

```diff
@@ -0,0 +1,48 @@
+; RUN: llc -mtriple=amdgcn -mcpu=gfx1100 -mattr=+real-true16 -verify-machineinstrs < %s 2>&1 | FileCheck -enable-var-scope -check-prefixes=GFX11 %s
```

Comment:
Why do these need to be separate files? Also don't need -verify-machineinstrs, or to redirect stderr